commerce-product-service/src/main/java/com/liferay/commerce/product/internal/util/CPVersionContributorRegistryImpl.java

```diff
@@ -67,9 +68,15 @@ public CPVersionContributor getCPVersionContributor(String key) {
 			cpVersionContributorServiceWrappers = ListUtil.fromCollection(
```

Comment:
We probably don't need to turn this into a list before iterating over it now, yeah? Saves us an object and a bit of unnecessary processing.